charms/kfp-persistence/metadata.yaml

```diff
@@ -0,0 +1,24 @@
+name: kfp-persistence
+display-name: Kubeflow Pipelines Persistence
+summary: Reusable end-to-end ML workflows built using the Kubeflow Pipelines SDK
+description: |
+  Machine learning (ML) toolkit that is dedicated to making deployments
+  of ML workflows on Kubernetes simple, portable, and scalable.
+maintainers:
+  - "Dominik Fleischmann <dominik.fleischmann@canonical.com>"
+  - "Kenneth Koski <kenneth.koski@canonical.com>"
+tags: [ai, big-data, kubeflow, machine-learning, tensorflow]
+series: [kubernetes]
+resources:
+  oci-image:
+    type: oci-image
+    description: Backing OCI image
+    auto-fetch: true
+    upstream-source: gcr.io/ml-pipeline/persistenceagent:1.5.0-rc.0
+provides:
+  kfp-persistence:
```

Comment:
How is it providing this? There is no sign of it in the charm code/

Reply:
Whoops, removed